app/init.php

```diff
@@ -232,6 +232,8 @@
 // Usage metrics
 const METRIC_TEAMS = 'teams';
 const METRIC_USERS = 'users';
+const METRIC_DATABASE_API_READ  = 'database.api.events.read';
```

Comment:
1. Lets also have per database metrics
2. We don't really need either `api` (all stats are technically from the API, not need to be explicit about it) or `events` (used at Appwrite all for triggering functions and webhooks)
3. In other places we use `databases` and not `database`

Maybe we can instead do: `databases.operations.reads` and `databases.operations.writes`

Reply:
We can also move this lines closer to other database related stuff we have on this list.